Builds/Test.py

```diff
@@ -61,6 +66,41 @@ def powerset(iterable):
 IS_WINDOWS = platform.system().lower() == 'windows'
 IS_OS_X = platform.system().lower() == 'darwin'
 
+# CMake
+
+CMAKE_UNITY_TARGETS = ['rippled', 'rippled_classic']
+CMAKE_NONUNITY_TARGETS = ['rippled', 'rippled_unity']
+if IS_WINDOWS:
+    CMAKE_UNITY_CONFIGS = ['Debug', 'Release']
+    CMAKE_NONUNITY_CONFIGS = ['DebugClassic', 'ReleaseClassic']
+else:
+    CMAKE_UNITY_CONFIGS = []
+    CMAKE_NONUNITY_CONFIGS = []
+CMAKE_UNITY_COMBOS = { '' : [CMAKE_UNITY_TARGETS, CMAKE_UNITY_CONFIGS],
+    '.nounity' : [CMAKE_NONUNITY_TARGETS, CMAKE_NONUNITY_CONFIGS] }
```

Comment:
I decided not to do this when I wrote the ALL_TARGETS list comprehension below, because I found it harder to read and understand the code if written like this for very little benefit (new targets rarely get added).

Please consider also simply explicitly listing the combinations instead of combining several lists with very few elements.